src/registrar/views/domain_requests_json.py

```diff
@@ -12,14 +12,55 @@ def get_domain_requests_json(request):
     """Given the current request,
     get all domain requests that are associated with the request user and exclude the APPROVED ones"""
```

Comment:
I think we should update the main comment..

```suggestion
    get all domain requests that are associated with the request user and exclude the APPROVED ones.
    If we are on the portfolio requests page, limit the response to only those requests associated with
    the given portfolio."""
```